locales/en.default.schema.json

```diff
@@ -658,7 +658,7 @@
       "settings": {
         "custom_liquid": {
           "label": "Liquid code",
-          "info": "Add app snippets or other code to create advanced customizations."
+          "info": "Add app snippets or other code to create advanced customizations. [Learn more](https://shopify.dev/docs/api/liquid)"
```

Comment:
The icon isn't defined at the theme level 🤔 So we would need to work with the folks that do editor work. Maybe @kmeleta would know 😅

Reply:
The offsite icon was introduced with the online-store-editor and can't be changed in dawn. We would have to create a separate issue/ticket for this @kimberlyoleiro